towerlib/entities/job.py

```diff
@@ -649,8 +657,68 @@ def job_events(self):
                              primary_match_field='name',
                              url=url)
 
+    def monitor(self, timeout=None, interval=0.25):
+        """Monitor the job execution events.
+
+        Returns:
+            string: The job status of the job
+
+        """
+        next_line = 0
+        started = time.time()
+        while self.status not in ["running"]:
```

Comment:
Wouldn't this block forever if there is not timeout and the status is failed for example?

Reply:
You're right. Will need to expand this to support the following status: ("error", "cancelled")